fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/tokenizer/multi_tokenizer.py

```diff
@@ -0,0 +1,1304 @@
+        stride: Optional[int] = 0,
+        strategy: Optional[str] = "longest_first",
+        direction: Optional[str] = "right",
+    ) -> None:
+        """
+        Enable truncation
+
+        Args:
+            max_length (:obj:`int`):
+                The max length at which to truncate
+
+            stride (:obj:`int`, `optional`):
+                The length of the previous first sequence to be included in the overflowing
+                sequence
+
+            strategy (:obj:`str`, `optional`, defaults to :obj:`longest_first`):
+                The strategy used to truncation. Can be one of ``longest_first``, ``only_first`` or
+                ``only_second``.
+
+            direction (:obj:`str`, defaults to :obj:`right`):
+                Truncate direction
+        """
+        raise Exception("Not implemented")
+
+    def encode_batch(
+        self,
+        input: List,
+        is_pretokenized: Optional[bool] = False,
+        add_special_tokens: Optional[bool] = True,
+    ) -> List:
+        """
+        Encode the given batch of inputs. This method accept both raw text sequences
+        as well as already pre-tokenized sequences.
+
+        Example:
+            Here are some examples of the inputs that are accepted::
+
+                encode_batch([
+                    "A single sequence",
+                    ("A tuple with a sequence", "And its pair"),
+                    [ "A", "pre", "tokenized", "sequence" ],
+                    ([ "A", "pre", "tokenized", "sequence" ], "And its pair")
+                ])
+
+        Args:
+            input (A :obj:`List`/:obj:`Tuple` of :obj:`~tokenizers.EncodeInput`):
+                A list of single sequences or pair sequences to encode. Each sequence
+                can be either raw text or pre-tokenized, according to the ``is_pretokenized``
+                argument:
+
+                - If ``is_pretokenized=False``: :class:`~tokenizers.TextEncodeInput`
+                - If ``is_pretokenized=True``: :class:`~tokenizers.PreTokenizedEncodeInput`
+
+            is_pretokenized (:obj:`bool`, defaults to :obj:`False`):
+                Whether the input is already pre-tokenized
+
+            add_special_tokens (:obj:`bool`, defaults to :obj:`True`):
+                Whether to add the special tokens
+
+        Returns:
+            A :obj:`List` of :class:`~tokenizers.Encoding`: The encoded batch
+
+        """
+        raise Exception("Not implemented")
+
+    @staticmethod
+    def from_buffer(buffer: object) -> object:
+        """
+        Instantiate a new :class:`~tokenizers.Tokenizer` from the given buffer.
+
+        Args:
+            buffer (:obj:`bytes`):
+                A buffer containing a previously serialized :class:`~tokenizers.Tokenizer`
+
+        Returns:
+            :class:`~tokenizers.Tokenizer`: The new tokenizer
+        """
+        raise Exception("Not implemented")
+
+    @staticmethod
+    def from_file(path: str) -> object:
```

Comment:
it would be nice if we can implement it.

Reply:
Can't see which method you mean. Is it encode_batch, from_buffer (not relevant) or from_file (also not relevant, since we save multiple files, and load them with load_from_jsons)?